src/client/layouts/Main.tsx

```diff
@@ -21,6 +21,7 @@ interface MainLayoutProps {
   pageTitle?: string;
   successOverride?: string;
   errorOverride?: string;
+  errorContext?: React.ReactNode | string;
```

Comment:
`ReactNode` can be a `string`, so don't need the explicit `string` type here, see here: https://stackoverflow.com/a/58123882.

```suggestion
  errorContext?: React.ReactNode;
```